airbyte-integrations/connectors/source-sftp-bulk/metadata.yaml

```diff
@@ -22,8 +24,13 @@ data:
     oss:
       enabled: true
   releaseStage: alpha
+  releases:
+    breakingChanges:
+      1.0.0:
+        message: "This upgrade brings migration to File-Based CDK"
+        upgradeDeadline: "2024-04-08"
```

Comment:
Is this meant to be immediate or should we switch the date to two weeks in the future?

Reply:
updated